app/src/main/java/ithaca_transit/android/cornellappdev/com/ithaca_transit/Models/Place.java

```diff
@@ -15,7 +15,8 @@
     private String placeID;
     private String type;
 
-    public Place(String detail, Double latitude, Double longitude, String name, String placeID, String type) {
+    public Place(String detail, Double latitude, Double longitude, String name, String placeID,
```

Comment:
pretty sure it's more idiomatic to have a parameter per line if it's over a line

Reply:
Ah, that was the auto-linter. I'll make those changes now.